src/Raven.Client/Replication/Messages/ReplicationLatestEtag.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Raven.Client.Replication.Messages
+{
+    public class ReplicationLatestEtag
```

Comment:
There is no mention of `ReplicationLatestEtag` in the actual content. You probably better of naming it `ReplicationLatestEtagRequest`